docs/advanced-topics/snapshot-and-doi.rst

```diff
@@ -0,0 +1,101 @@
+Creating Snapshots and Requesting DOIs
+=============================================
+
+Introduction
+------------------
+Starting in Dockstore 1.7.0, users can create snapshots and Digital
+Object Identifiers (DOIs) for their workflows. The ‘Snapshot’ and
+‘Request DOI’ actions can be found on the versions tab of any published
```

Comment:
The snapshot action is available for me for non-published (hosted, in case that's relevant) workflows